nipype/algorithms/confounds.py

```diff
@@ -664,7 +662,7 @@ def _run_interface(self, runtime):
             )
 
         components, filter_basis, metadata = compute_noise_components(
-            imgseries.get_data(),
+            imgseries.get_fdata(dtype=np.float32),
```

Comment:
Please specifically verify that the desired behavior here would be to compute noise components on float32 data... float64 would be very large, but this data is presumably floating point, so using the dataobj interface seems needlessly cumbersome.